crypto/cms/cms_lcl.h

```diff
@@ -419,6 +419,10 @@ int cms_RecipientInfo_pwri_crypt(const CMS_ContentInfo *cms, CMS_RecipientInfo *
 /* SignerInfo routines */
 int CMS_si_check_attributes(const CMS_SignerInfo *si);
 
+/* ESS routines */
+int CMS_add1_signing_cert(CMS_SignerInfo *si, ESS_SIGNING_CERT *sc);
```

Comment:
use lowercase cms now that they are local e.g. cms_add1_signing_cert

Reply:
done !